src/components/Categories/CategoriesPresentation.js

```diff
@@ -40,18 +40,16 @@ function CategoriesPresentation({ categories, kids, sitename }) {
                   <ul className="grid grid-cols-2 gap-3 md:grid-cols-3 md:gap-6 lg:grid-cols-3 xl:grid-cols-4 xl:gap-8">
                     {categories?.length > 0 ? (
                       categories.map((category) => (
-                        <li
+                        <Link
```

Comment:
This may work in some browsers, but it isn't valid html. Only allowed children of a ul element are list items

Reply:
Good call, thanks. I have reverted to the previous code, allowed the parent div to use the full height and width of the box, and added padding to recenter the text/image.